package.json

```diff
@@ -47,14 +47,16 @@
   },
   "scripts": {
     "deduplicate": "node scripts/deduplicate.js",
+    "jsonlint": "node scripts/jsonlint.js",
     "hoist": "lerna bootstrap --hoist",
     "bootstrap": "lerna bootstrap",
     "build": "lerna run build --stream",
     "start": "lerna run start --parallel",
     "prettier": "node ./scripts/prettier.js",
     "test": "lerna run test --parallel",
-    "lint": "eslint . --cache --report-unused-disable-directives --ext .js,.ts,.tsx",
-    "lint:ci": "eslint . --report-unused-disable-directives --ext .js,.ts,.tsx"
+    "lint": "yarn eslint && yarn jsonlint",
```

Comment:
I need to add stylelint later on, adding it to the list.

Reply:
I've added stylelint in xgrid-modules, maybe we can scale it to the repo... 🤔

Reply:
Maybe, the reason I didn't spend time on it, so far, is because it's still unclear if we will need it.

1. We need it now
2. Short term, we won't with core's CSS-in-JS solution (eslint is enough to lint the styles, not need for stylelint)
3. Long term, we might switch to the CSS syntax.